### PR TITLE
Added -yes flag to run non-interactively.

### PR DESCRIPTION
**Pull request by navossoc**
Not sure if is really needed but it make scripting easier.

Sure we can do a: 
```echo y | gothanks```

Anyhow, it's your call :)

### CODE REVIEW

**Comment by psampaz** [comment]
thank you @navossoc!

File: main.go [hunk 1]
```diff
@@ -35,6 +35,7 @@ import (
 
 func main() {
 	githubToken := flag.String("github-token", os.Getenv("GITHUB_TOKEN"), "Github access token")
+	assumeYes := flag.Bool("yes", false, "Automatic yes to prompts and run non-interactively.")
```
Comment (psampaz) [inline main.go]: Please rename this flag from yes to y in order to match the  "Press y to continue or n to abort"

**Comment by psampaz** [comment]
Please also document this flag in the Usage section of the README.md

File: main.go (hunk 1 above)
Comment (navossoc) [reply main.go]: Done...

**Comment by navossoc** [comment]
I also did a rebase since we updated the `README.md` on the other pull request.
Now you can just merge and squash all the changes easily.

**Comment by psampaz** [comment]
Excellent!

One more thing before I merge this:

Please update the Changelog with 

```
## [0.2.0] - 2019-11-17
### Added
- Run it on non-interactive environments using `-y` flag
```

**Comment by navossoc** [comment]
Sure.

**Pull request merged**